src/Physics/physicsHelper.ts

```diff
@@ -694,14 +716,107 @@ export class PhysicsVortexEvent {
 
 }
 
+/**
+ * Options fot the radial explosion event
+ * @see https://doc.babylonjs.com/how_to/using_the_physics_engine
+ */
+export class PhysicsRadialExplosionEventOptions {
+    /**
+     * The radius of the sphere for the radial explosion.
+     */
+    radius: number = 5;
+
+    /**
+     * The strenth of the explosion.
+     */
+    strength: number = 10;
+
+    /**
+     * The strenght of the force in correspondence to the distance of the affected object
+     */
+    falloff: PhysicsRadialImpulseFalloff = PhysicsRadialImpulseFalloff.Constant;
+
+    /**
+     * Sphere options for the radial explosion.
+     */
+    sphere: { segments: number, diameter: number } = { segments: 32, diameter: 1 };
+}
+
+/**
+ * Options fot the updraft event
+ * @see https://doc.babylonjs.com/how_to/using_the_physics_engine
+ */
+export class PhysicsUpdraftEventOptions {
+    /**
+     * The radius of the cylinder for the vortex
+     */
+    radius: number = 5;
+
+    /**
+     * The strenth of the updraft.
+     */
+    strength: number = 10;
+
+    /**
+     * The height of the cylinder for the updraft.
+     */
+    height: number = 10;
+
+    /**
+     * The mode for the the updraft.
+     */
+    updraftMode: PhysicsUpdraftMode = PhysicsUpdraftMode.Center;
+}
+
+/**
+ * Options fot the vortex event
+ * @see https://doc.babylonjs.com/how_to/using_the_physics_engine

```

Comment:
is that the right link?

Reply:
I think so? Was copy/pasting from other classes. Still, need to update the documentation there soon, when everything else is now fine.

Reply:
no worries :) I'm fine to makr then private then

Reply:
I think you should point to this one: https://doc.babylonjs.com/how_to/particlehelper

Reply:
Particle helper? For what? The whole `PhysicsHelper` does not do anything on the graphics part. Only physics.
So that's what it does: https://playground.babylonjs.com/index.html#TVUDC1#4

You'll still need to do the visual effect manually.

Reply:
lol bad copy /paste
https://doc.babylonjs.com/how_to/using_the_physics_engine#further-functionality-of-the-impostor-class

Reply:
Oh, well, that one makes more sense now :)
Sure! Updated the documentation stuff in the helper!